### PR TITLE
Add Ember 4.4 LTS to addon blueprint, remove 3.24

### PR DESCRIPTION
**Pull request by simonihmig**

### CODE REVIEW

**Comment by simonihmig** [comment]
I can't see anything that would explain the one failing windows job... except flakiness?

File: blueprints/addon/files/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 ## Compatibility
 
-* Ember.js v3.24 or above
+* Ember.js v3.28 or above
 * Ember CLI v3.24 or above
```
Comment (bertdeblock) [inline blueprints/addon/files/README.md:9]:
```suggestion
* Ember CLI v3.28 or above
```

AFAIK, these can be in sync. Some more info can be found here: https://github.com/ember-cli/ember-cli/pull/9491.

**Comment by bertdeblock** [comment]
> I can't see anything that would explain the one failing windows job... except flakiness?

flakiness indeed, feel free to ping me on Discord or here to restart failed jobs.

File: blueprints/addon/files/README.md (hunk 1 above)
Comment (simonihmig) [reply blueprints/addon/files/README.md]: Makes sense, committed your change. I was used to see the CLI version be far behind ember-source in the past...

**Comment by simonihmig** [comment]
@bertdeblock I addressed your feedback, and CI also seems happy now! :tada:

**Comment by kategengler** [comment]
Thank you!

**Pull request merged**